development/docker-compose.yml

```diff
@@ -4,33 +4,24 @@ services:
   nautobot:
     build:
       context: ../
-      dockerfile: development/Dockerfile
-    image: "nautobot/nautobot-py${PYTHON_VER}"
-    command: "nautobot-server runserver 0.0.0.0:8000 --insecure"
+      dockerfile: docker/Dockerfile
+      target: dev
+    image: "nautobot/nautobot-py${PYTHON_VER}-dev"
```

Comment:
I think the image name needs to be `networktocode/nautobot` to be able to publish it to dockerhub today
and we should use the tag to capture the version of nautobot : `networktocode/nautobot:1.0.0`

Not sure if we need to support multiple versions of python right way, if we do, I would recommend to create a tag that has both the version of nautobot and the version of python, something like that:
- networktocode/nautobot:1.0.0-3.7
- networktocode/nautobot:1.0.0-3.8